src/AspNetCoreModuleV2/OutOfProcessRequestHandler/forwardinghandler.cpp

```diff
@@ -140,7 +141,7 @@ FORWARDING_HANDLER::OnExecuteRequestHandler()
     }
 
     hr = pApplication->GetProcess(&pServerProcess);
-    if (FAILED(hr))
+    if (FAILED_LOG(hr))
```

Comment:
Add a macro for goto failure?

Reply:
Didn't want to go through this entire huge file in this PR anyway. Just making sure we observe if something fails.